lib/membrane_ffmpeg_swresample_plugin/converter.ex

```diff
@@ -122,7 +122,13 @@ defmodule Membrane.FFmpeg.SWResample.Converter do
   end
 
   @impl true
-  def handle_buffer(:input, %Buffer{payload: payload}, _ctx, state) do
+  def handle_buffer(:input, %Buffer{payload: payload, pts: pts}, _ctx, state) do
+    state =
+      state
+      |> Map.merge(%{
+        current_pts: pts
+      })
```

Comment:
```suggestion
|> Map.put(:current_pts, pts)
```

Reply:
@bartkrak actually, the right approach would be to add this field in `handle_init` and do `%{state | current_pts: pts}`. Adding fields to state in various callbacks makes it hard to follow.